dbscripts/xml/ojs_schema.xml

```diff
@@ -507,9 +504,6 @@
 			<DEFAULT VALUE="1"/>
 		</field>
 		<descr>Submissions</descr>
-		<index name="submissions_user_id">
-			<col>user_id</col>
-		</index>
```

Comment:
Don't we need to know who uploaded the article in the first place? This can be still retrieved using the article log?